### ISSUE TITLE
Update 12/31/2024

### ISSUE THREAD

**Issue by Crpeekaboo** [Vae2009/ConROC#16]
Overall update in testing Hunter, Priest, Warrior, Rogue, Pally, Warlock and Mage.

(1) Warlock work super well
(2) Pally work really nice, except for the error for calling for a weapon item Seraph's Strike. Not sure why as it's a weapon. I don't have this weapon.
(3) Hunter works great.
(4) Priest works great.
(5) Warrior works well, but I would add to call for Victory Rush as it's a heal for Warriors when active soecial when using charge if possible.. Other than that works great.
(6) Druid works great.
(7) Mage needs work. Will not call for spell to be used...getting no errors.
(8) Rogue has major errors. Will freeze up screen and sometime disconnect.

I Listed all I found on here and include screenshot best I could.

I know this is a work in process and it's starting to look really great.

Thanks

**Comment by Starlynk1** [comment]
Main ConRoc error
```
2x ConROC/buttons.lua:470: attempt to index global 'ConRO' (a nil value)
[string "@ConROC/buttons.lua"]:470: in function `DisplayWindowFrame'
[string "@ConROC/core.lua"]:904: in function <ConROC/core.lua:899>
[string "=[C]"]: ?
[string "@Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:66: in function <Ace3/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:619: in function <Ace3/AceAddon-3.0/AceAddon-3.0.lua:611>
```

**Comment by Starlynk1** [comment]
Update: Tried different things, reinstalled, removed all folders and reinstalled.

Finally went into the addon options and reset position and reset settings. This fixed the errors. Looks like the changes have an impact on the saved settings and positioning.

**Comment by Starlynk1** [comment]
In the options, I've narrowed it down the the Show Keybind checkbox. Disabling this causes the errors listed above. Enabling it fixes the errors. So It is possible that the Reset Position and Reset Settings is not needed to fix the error. Just make sure Show Keybind is enabled.

**Comment by Vae2009** [comment]
Found the error here is do to a missing C in the code. Will be fixed in v1.15.21.
